### PR TITLE
fix issue of aliased variable

### LINKED ISSUE #281: predict() returns error message if no outcome data at certain visit for certain group
To do:
- [x] Add reprex as new integration test
- [x] Fix bug:
	- [x] return understandable warning message
- [x] Ensure all tests pass
- [x] Note: no NEWS update needed as we don't have a release with predict yet.

Report: I am trying to use the `predict()` function from a `mmrm` fit object. In my case, I have no outcome data at certain visits for at least one treatment group. I have no problem in fitting the model and extracting marginal means using `emmeans`, but I am having an issue with predict(). Please find below an example:

```r

# Add fake visit only for one treatment group
new_fev_data <- rbind(
  fev_data,
  fev_data %>% 
    filter(ARMCD == fev_data$ARMCD[1], AVISIT == "VIS1") %>% 
    mutate(AVISIT = "VIS5", FEV1 = rnorm(n(), mean = 45, sd = 5))
)

# Fit mmrm
fit <- mmrm(
  formula = FEV1 ~ ARMCD * AVISIT + us(AVISIT | USUBJID),
  data = new_fev_data
)

# Predict
predict(fit, fit$data)
```

Error message:

```r
Error in h_get_prediction(tmb_data, object$theta_est, object$beta_est,  : 
  Assertion on 'beta' failed: Must have length 10, but has length 9.
```

**R session info**

R version 4.2.1
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.5 LTS

### PR DESCRIPTION
**Pull request by clarkliming**
close #281

### CODE REVIEW

**Comment by clarkliming** [comment]
@danielinteractive sorry need to bother you again. too late make mistakes. sorry 😢

**Comment by danielinteractive** [comment]
No worries @clarkliming :-)

File: R/tmb-methods.R [hunk 1]
```diff
@@ -82,6 +82,14 @@ predict.mmrm_tmb <- function(
     allow_na_response = TRUE,
     drop_levels = FALSE
   )
+  if (any(object$tmb_data$x_cols_aliased)) {
+    warning(
+      "In fitted object there are co-linear variables and dropped, ",
```
[the remark this answers is not in the document]
Comment (danielinteractive) [reply R/tmb-methods.R:87]: and similar for the test

[the remark this answers is not in the document]
Comment (clarkliming) [reply R/tmb-methods.R]: updated

**Review by danielinteractive — APPROVED** [review]

**Pull request merged**